migration.md

```diff
@@ -1,118 +1,124 @@
-
-
-```
-celery-opentracing
-django-opentracing
-elasticsearch-opentracing
-Flask-OpenTracing
-sfx-jaeger-client
-dbapi-opentracing
-pymongo-opentracing
-dbapi-opentracing
-redis-opentracing
-requests-opentracing
-signalfx-tracing
-tornado-opentracing
-``` 
-
-In addition to these, you should remove any other opentracing instrumentations you might be using.
-
-
-## Install splunk-opentelemetry
-
-Now that we've removed the OpenTracing based SignalFx Python library (signalfx-python-tracing), it is time to install the next-gen
-OpenTelemetry based Python library called `splunk-opentelemetry`.
-
-### Install splunk-opentelemtry
-
-#### Using pip
-```
-pip install splunk-opentelemetry
-
-```
-You should also add `splunk-opentelemetry` to your requirements.txt file if you are using one.
-
-#### Using poetry
-```
-poetry add splunk-opentelemetry
-```
-
-### Installing dependencies
-
-Just like signalfx-python-tracing, splunk-openetelemetry also ships with a bootstrap command that automatically installs any
-instrumentation packages your project could benefit from. The following command will detect and auto-install any instrumentations
-for any packages it finds in your Python environment.
-
-```
-splk-py-trace-bootstrap
-```
-
-You can integrate this command in your build/deployment process so instrumentations are installed every time your project is deployed
-or you can run this once during development and add all the installed packages as dependencies. The command also supports printing out
-the list of instrumentation packages instead of automatically installing them. For example,
-
-```
-splk-py-trace-bootstrap -a=requirements
-
-opentelemetry-instrumentation-falcon>=0.13b0
-opentelemetry-instrumentation-requests>=0.8b0
-opentelemetry-instrumentation-sqlite3>=0.11b0
-opentelemetry-exporter-zipkin>=0.14b0
-```
-
-## Migrate configuration
-
-### General Config
-
-You must rename the following config environment variables
-
-| Old Environment Variable           | New Environment Variable             |
-| ---------------------------------- | ------------------------------------ |
-| SIGNALFX_SERVICE_NAME              | SPLK_SERVICE_NAME                    |
-| SIGNALFX_ENDPOINT_URL              | OTEL_EXPORTER_ZIPKIN_ENDPOINT        |
-| SIGNALFX_RECORDED_VALUE_MAX_LENGTH | SPLK_MAX_ATTR_LENGTH                 |
-
-
-### Django
-
-#### Remove `signalfx_tracing` from INSTALLED_APPS
-
-splunk-opentelemtry no longer ships a Django app and does not require you to add anything to your `INSTALLED_APPS` or any other
-setting from settings.py
-
-#### Set `DJANGO_SETTINGS_MODULE` environment variable
-
-splunk-opentelemetry requires `DJANGO_SETTINGS_MODULE` environment to be set before it can auto-instrument a Django project. This
-should be the same as the value set in your manage.py or wsgi.py app and should be a valid Python import path.
-
-## Known issues
```

Comment:
This section had been removed. Should be added bank

Reply:
- OpenTelemetry's psycopg2 has a bug that prevents the driver from functioning correctly. Thus it has been disabled temporarily in this package.
-- this isn't specific to the migration.
- Does not yet support sending spans directly to the SignalFx backend and requires the SignalFx Smart Agent or OpenTelemetry collector.
-- this is no longer true.
- Does not yet support enabling or disabled specific instrumentations.
-- this is not a known issue of the migration or the package.
- Does not yet support tracing and log correlation.
-- this is not a known issue of the migration of the package.

Reply:
Other than the second point, all of them are known issues and relate directly to migration. These are features that work with signalfx-python-tracing but do not work with splunk-otel-python so migrating will cause these regressions and people should know about this. It's very important to call these out so users aren't surprised after completing the migration only to find out these things not working.